changelog/v1.18.0-rc2/fix-us-status-flicker.yaml

```diff
@@ -0,0 +1,16 @@
+changelog:
+  - type: NON_USER_FACING
+    description: >-
+      Fix flicker on Upstream status when kube gw syncer rejects a resource and no edge proxies are present
+    issueLink: https://github.com/solo-io/solo-projects/issues/7243
+    resolvesIssue: true
+  - type: NON_USER_FACING
+    description: >-
+      Fix missing status on krt objects resulting in continuous status updates (and webhook hits)
+    issueLink: https://github.com/solo-io/solo-projects/issues/7257
+    resolvesIssue: true
+  - type: BREAKING_CHANGE
+    description: >-
+      Upstreams and UpstreamGroups no longer get Accepted status by default. If they have not gone through translation they will have an empty status field.
```

Comment:
Iff using kube gateways right?

Reply:
in all cases now.
i.e. the scenario described in https://github.com/solo-io/gloo/issues/10401 will be resolved regardless of kube gateways